src/nspawn/nspawn.c

```diff
@@ -2523,6 +2524,23 @@ static int on_orderly_shutdown(sd_event_source *s, const struct signalfd_siginfo
         return 0;
 }
 
+static int on_sigchld(sd_event_source *s, const struct signalfd_siginfo *si, void *userdata) {
+        pid_t pid;
+
+        // Because we set PR_SET_CHILD_REAPER, orphan processes may be
+        // reparented to us.
```

Comment:
Nit:
> - We prefer /* comments */ over // comments, please. This is not C++, after
  all. (Yes we know that C99 supports both kinds of comments, but still,
  please!)

https://github.com/systemd/systemd/blob/e28c7cd0665364bb910fe2cead882623c23c28ac/CODING_STYLE#L4